wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-2/default.yaml

```diff
@@ -0,0 +1,377 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As mentioned earliert, it might be better to have just one pattern with a MySQL backend, similar to what's being done in AM Analytics [1].

[1] - https://github.com/wso2/puppet-apim/blob/master/wso2am_analytics/hieradata/dev/wso2/wso2am_analytics/pattern-1/default.yaml